src/blocks/embed-markdown/render.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * PHP file to use when rendering the block type on the server to show on the front end.
+ *
+ * The following variables are exposed to the file:
+ *     $attributes (array): The block attributes.
+ *     $content (string): The block default content.
+ *     $block (WP_Block): The block instance.
+ *
+ * @see https://github.com/WordPress/gutenberg/blob/trunk/docs/reference-guides/block-api/block-metadata.md#render
+ * @package labs-plugin-blocks
+ */
+
+// Add ID attribute if anchor is set.
+if ( ! empty( $attributes['anchor'] ) ) {
+	$block_wrapper_attributes['id'] = esc_attr( $attributes['anchor'] );
+}
+
+// Convert GitHub webapp URLs to raw markdown URLs.
+$md_url = esc_url_raw( $attributes['mdURL'] );
+$raw    = str_replace( '/github.com/', '/raw.githubusercontent.com/', $md_url );
+$raw    = str_replace( '/blob/', '/', $raw );
+
+$response = wp_remote_get( $raw );
+
+if ( is_array( $response ) && ( 200 === wp_remote_retrieve_response_code( $response ) ) && ( ! is_wp_error( $response ) ) ) {
+	?>
+<article <?php echo $attributes['anchor'] ? 'id="' . esc_attr( $attributes['anchor'] ) . '"' : ''; ?> <?php echo wp_kses_data( get_block_wrapper_attributes() ); ?>>
```

Comment:
```suggestion
<article <?php echo wp_kses_data( get_block_wrapper_attributes( $block_wrapper_attributes ) ); ?>>
```

Reply:
I would suggest letting `get_block_wrapper_attributes()` handle outputting any of the parameters on the wrapping element. In some of the BU blocks I noticed some oddities sometimes when I was adding the anchor ID manually where things were not outputting correctly in the rendered HTML. When I switched it to let `get_block_wrapper_attributes()` handle it, things seemed to be better.

Reply:
I think I might have read that in newer versions of WP `get_block_wrapper_attributes()` handles more parameters by default as well. They are trying to make it complementary to `useBlockProps` in the editor. I think 5.8 is just a bit behind.